model/src/test/kotlin/VulnerabilityTest.kt

```diff
@@ -27,43 +27,40 @@ import io.kotest.matchers.shouldBe
 
 import java.lang.IllegalArgumentException
 
-import org.ossreviewtoolkit.model.Vulnerability.Cvss2Rating
-import org.ossreviewtoolkit.model.Vulnerability.Cvss3Rating
-
 class VulnerabilityTest : StringSpec({
     "The CVSS 2 rating should be correct for a given score" {
-        Cvss2Rating.fromScore(-0.1f) should beNull()
+        VulnerabilityReference.Cvss2Rating.fromScore(-0.1f) should beNull()
```

Comment:
Please just change the imports so all code can stay the same, for a smaller diff.